### PR TITLE
Python プログラミング入門

### PR DESCRIPTION
**Pull request by Shunuk**
## What does this PR do?
Add resource(s)

### Description
I added free-programming book from "University of Tokyo Mathematics and Information Education Research Center"
### Why is this valuable (or not)?
It is the book that teaching about python by 322 pages
### How do we know it's really free?
It is on the google platform and you don't need to spend money to read it and it's written by known as the smartest university in Japan.  
### For book lists, is it a book? For course lists, is it a course? etc.
Book
## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-ja.md [hunk 1]
```diff
@@ -596,7 +596,7 @@
 * [プログラミング演習 Python 2019](http://hdl.handle.net/2433/245698) - 喜多一 (PDF)
 * [みんなのPython Webアプリ編](https://coreblog.org/ats/stuff/minpy_web) - 柴田淳
 * [機械学習の Python との出会い (Machine Learning Meets Python)](https://www.kamishima.net/mlmpyja) - 神嶌敏弘 [PDF](https://www.kamishima.net/archive/mlmpyja.pdf), [EPUB](https://www.kamishima.net/archive/mlmpyja.epub)
-
+* [Python プログラミング入門](https://teams.microsoft.com/l/message/48:notes/1697129049401?context=%7B%22contextType%22%3A%22chat%22%7D) - 東京大学 数理・情報教育研究センター
```
Comment (3R1Dev) [inline books/free-programming-books-ja.md:599]: This requires someone to either be logged in to their Microsoft account or create one. I cannot verify the content, and it also appears to be notes to or in a chat based on the URL.

Are you able to provide the original link? Or a link that is not behind a login?

File: courses/free-courses-ja.md [hunk 2]
```diff
@@ -11,4 +11,6 @@
 
 ### Scratch
 
+* [How to make Mochi Master Game in scratch](https://www.youtube.com/watch?v=1cCth98Y2Lo) - Coding With Chris
```
Comment (3R1Dev) [inline courses/free-courses-ja.md:14]: This is a good tutorial but the content is in English.

File: books/free-programming-books-ja.md (hunk 1 above)
Comment (Shunuk) [reply books/free-programming-books-ja.md:599]: Pythonプログラミング入門  ( https://utokyo-ipp.github.io/?shem=iosie ) 
I think I attached the wrong link? But thank you to let me know it.

File: courses/free-courses-ja.md (hunk 2 above)
Comment (Shunuk) [reply courses/free-courses-ja.md:14]: I'm wondering about video's category, is video is a course or some else category?

**Comment by eshellman** [comment]
I agree with @3R1Dev  - not suitable for this list

**Pull request closed without merging**